weblab/entities/forms.py

```diff
@@ -57,7 +58,7 @@ class EntityVersionForm(forms.Form):
     )
     tag = forms.CharField(
         help_text='Optional short label for this version',
-        required=False)
+        required=False, validators=[RegexValidator(r'[-_A-Za-z0-9]', 'Please enter a valid tag name.')])
```

Comment:
Just a stylistic thing: it's best practice to have each arg on a separate line when they don't all fit on one.
```suggestion
        required=False,
        validators=[RegexValidator(r'[-_A-Za-z0-9]', 'Please enter a valid tag name.')])
```

Reply:
done

Reply:
The other occurrence is done, but you missed this one :)

Reply:
sorry, changed that now